Networking/WebSockets/WebSocketImpl.cc

```diff
@@ -97,13 +97,33 @@ namespace litecore { namespace websocket {
         startResponseTimer(chrono::seconds(kConnectTimeoutSecs));
     }
 
+    void WebSocketImpl::gotTLSCertificate(slice certData) {
+        std::lock_guard<std::mutex> lock(_mutex);
+
+        if (_closed) {
+            warn("WebSocket already closed, ignoring onWebSocketGotTLSCertificate...");
```

Comment:
I don't think this is worth warning about, it's just a harmless race condition.

Reply:
Removed the warnings in `gotTLSCertificate` and `gotHTTPResponse`.